Scripts/Tools/Tool.cs

```diff
@@ -62,12 +65,16 @@ public void SetSelection(GameObject selectedGameObject, GameObject[] selectedGam
 
 			// Make sure it's not null and that it isn't a prefab in the project
 			if(selectedGameObject != null
-#if UNITY_2018_2
+#if UNITY_2018_2_OR_NEWER || UNITY_2018_3
```

Comment:
```suggestion
#if UNITY_2018_2_OR_NEWER
```

I think you missed removing a " || UNITY_2018_3" statement. :)